### PR TITLE
fix(article): use locale formatted Modified (Last Updated) date

### PR DESCRIPTION
**Pull request by gertvdijk**
## Prerequisites

- [x] My Pull Request is filled against the `next` branch
- [x] All commits in my Pull Request conform to [Elegant Git commit Guidelines](https://elegant.oncrashreboot.com/git-commit-guidelines)

## Recommended Steps

- [ ] My patch adds a new feature, therefore I have also added a help article about it
- [ ] My patch changes Elegant behavior, therefore I have updated the help article to reflect this change
- [x] My commits are [signed](https://help.github.com/en/articles/signing-commits)

## Description

See #613

### CODE REVIEW

File: templates/_includes/last_updated.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 {% if delta > 0 %}
 <h4>Last Updated</h4>
 {% set day = article.modified.strftime('%d')|int %}
```
Comment (talha131) [inline templates/_includes/last_updated.html:9]: It seems we do not need day variable after your patch. Can you please remove it in the same commit?

Comment (gertvdijk) [reply templates/_includes/last_updated.html:9]: Ah, I see, thanks for pointing that out. :+1:  Let me have a look and get back to this.

**Comment by gertvdijk** [comment]
> When this code was written, Pelican did not have `locale_modified`.

Do you mean, when this related change to 'Published' date was written in 1da552c94775? It seems to be part of Pelican since 2013 (and you even commented there back in the days :smile: ): https://github.com/getpelican/pelican/commit/4b2fcb09a4bfd3f099013bbf64550e57b1e13e55

Anyways, I'll amend my commit with your valid comment. :smile:

File: templates/_includes/last_updated.html (hunk 1 above)
Comment (gertvdijk) [reply templates/_includes/last_updated.html]: done

**Comment by talha131** [comment]
> and you even commented there back in the days 😄 

Ah! Now I recall the whole thing. Yeah, it was an oversight on my part. Thank you for catching it.

**Comment by talha131** [comment]
@all-contributors please add @gertvdijk for bug.

**Pull request merged**